src/modules/scroll-window.ts

```diff
@@ -1,10 +1,16 @@
+import { headerHeight as headerHeightInPx } from '~/containers/search-controls';
+const headerHeight = parseInt(headerHeightInPx);
+
 export const scrollToTop: IO<void> = () => {
 	window.scrollTo(0, 0);
 };
 
-export const scrollToEl = (el: HTMLElement): IO<void> => (): void => {
-	const headerHeight = parseInt(window.getComputedStyle(document.body).getPropertyValue('--header-height'), 10);
+export const scrollToEl = (el: HTMLElement): void => {
```

Comment:
Haven't been in the codebase for a little while but I'd like to retain the previous type signature which expressed impurity. Might be that as a result an extra function call needs adding wherever it's being called.

Otherwise thanks for your contribution, looking good for me locally. :smile:

Reply:
Gotcha. I've committed a fix for that.